spark-indexer/src/main/java/org/uniprot/store/spark/indexer/proteome/converter/DatasetProteomeEntryConverter.java

```diff
@@ -19,26 +30,338 @@
  * @created 21/08/2020
  */
 public class DatasetProteomeEntryConverter implements Function<Row, ProteomeEntry>, Serializable {
-
     private static final long serialVersionUID = -6073762696467389831L;
+    public static final String UPID = "upid";
```

Comment:
regenerate serialVersionUID since the class has changed.. IntelliJ have option to do it.